scout/commands/update/panel.py

```diff
@@ -46,10 +60,47 @@ def panel(panel, version, update_date, update_version):
             LOG.warning(err)
             click.Abort()
 
+    # Any mintainer updates?
+
+    new_maintainer = None
+    if add_maintainer:
+        user_obj = adapter.user(user_id=add_maintainer)
+        if not user_obj:
+            # Check if maintainers exist in the user database
+            LOG.warning(
+                "Maintainer user id {0} does not exist in user database".format(
+                    add_maintainer
+                )
+            )
+            click.Abort()
+
+        new_maintainer = panel_obj.get("maintainer", [])
+        if add_maintainer in new_maintainer:
+            LOG.warning("User {} already in maintainer list.".format(add_maintainer))
+            click.Abort()
+
+        new_maintainer.append(add_maintainer)
+
+    if revoke_maintainer:
+        current_maintainers = panel_obj.get("maintainer", [])
+        try:
+            current_maintainers.remove(revoke_maintainer)
+            new_maintainer = current_maintainers
+        except ValueError:
+            LOG.warning(
+                (
+                    "Maintainer user id {} is not a maintainer for panel {}.".format(
+                        revoke_maintainer, panel
+                    )
+                    + " Current maintainers: {}.".format(current_maintainers)
```

Comment:
Also this could be in one string like
```python
LOG.warning(
    "Maintainer %s is not a maintainer for panel %s. Current maintainers: %s.", 
    revoke_maintainer, panel, current_maintainers
)
```

Reply:
100 chars..

Reply:
Ok like that then